🌐 Replaced hardcoded text to use translation

Tagaishi opened the pull request:
### Category
> Translation support

### Overview
> I've gone through the different files in widget and pages to try and find as many places where a hardcoded message was put instead of using translation and changed that.
> Since I went through the files I also corrected typos, removed unused dependencies and elements (not all of them, just when I noticed).
>This partly uses the new system for widget option select that when no label is issued, will automatically pull text from translation.

ajnart commented on public/locales/en/modules/media-requests-list.json:
```diff
@@ -19,6 +19,37 @@
+    "approved": "Request was approved!",
+    "declined": "Request was declined!"
+  },
+  "detail": {
+    "label": "Stats for nerds",
+    "id": "ID",
+    "device": "Device",
+    "video": {
+      "video":"Video",
+      "resolution": "Resolution",
+      "framerate": "Framerate",
+      "codec": "Video Codec"
+    },
+    "audio": {
+      "audio": "Audio",
+      "channels": "Audio Channels",
+      "codec": "Audio Codec"
+    },
+    "transcoding": {
+      "transcoding": "Transcoding",
+      "context": "Context",
+      "requested": "Hardware Encoding Requested",
+      "source": "Source Codec",
+      "target": "Target Codec"
+    }
```
We made the conscious decision to not translate that. It's for nerds they should understand English, this way less things to translate and less space on page load (please revert this)

Meierschlumpf replied:
We should provide those translations so the translators can select wheter or not they want to translate it. For the size it makes mostly no difference.

ajnart commented on public/locales/en/modules/media-requests-stats.json:
```diff
@@ -5,7 +5,11 @@
-        "label": "Direction of the layout."
+        "label": "Direction of the layout.",
+        "data":{
+          "row": "Horizontal",
+          "column": "Vertical"
```
I've seen it translated elsewhere. Put into common.json if it's common.

Meierschlumpf replied:
That property translations are used by the widget and so it is not possible to move it to the common.json

ajnart commented on public/locales/en/modules/media-server.json:
```diff
@@ -6,6 +6,7 @@
       "title": "Settings for media server widget"
     }
   },
+  "loading": "Homarr is loading streams...",
```
Just loading steams makes more sense. No need for "…" (anywhere)

Meierschlumpf replied:
I agree that we can move the dots to the actual string. The App name can be replaced with a variable

ajnart commented on public/locales/en/modules/torrents-status.json:
```diff
@@ -63,7 +63,8 @@
     "loading": {
-      "title": "Loading..."
+      "title": "Loading...",
+      "description": "Homarr is establishing a connection..."
```
Same thing, don't say "homarr" in translations. Just "establishing connection"

Meierschlumpf replied:
The App name can be replaced with a variable

ajnart commented on public/locales/en/modules/usenet.json:
```diff
@@ -31,7 +31,8 @@
     },
-    "paused": "Paused"
+    "paused": "Paused",
+    "notImplemented": "NOT IMPLEMENTED"
```
So no need to translate if it's not there lol. Just adds work for the ~25 languages we translate in for no reason.

Tagaishi commented:
After reading more on it, I noticed you can use timezones and gmt in dayjs too, so I switched everything over to it and removed moment to cut back on dependencies.
Fixed the locale in the calendar, locale language had to be imported into the calendar for it to work.
Finished handling Issue #764. All points there are addressed.
